pkg/test/case.go

```diff
@@ -304,6 +301,18 @@ func (t *Case) CollectTestStepFiles() (map[int64][]string, error) {
 	return testStepFiles, nil
 }
 
+// getIndexFromFile returns the index derived from fileName's prefix, ex. "01-foo.yaml" has index 1.
+func (t *Case) getIndexFromFile(fileName string) (int64, bool, error) {
+	matches := testStepRegex.FindStringSubmatch(fileName)
+	if len(matches) != 2 {
+		t.Logger.Log("Ignoring", fileName, "as it does not match file name regexp:", testStepRegex.String())
```

Comment:
I would personally prefer to return error from here and log in the method that's handling the error